uber/templates/macros.html

```diff
@@ -82,6 +82,36 @@
 {%- endmacro %}
 
 
+{% macro form_group_scaffolding(
+    label=False,
+    help=False,
+    is_required=False,
+    size=6,
+    form_class='',
+    static_wrap=False) -%}
```

Comment:
Yeah, you're probably right. It does make it much more convenient.

Here's another thing we haven't discussed – but I'm going to quibble about anyway – for HTML I prefer to use 2 spaces for indention. Generally, 4 spaces in python helps visually differentiate blocks. But by its nature, elements in HTML tend to be much more deeply nested than code blocks in Python. Using 2 spaces instead of 4 helps prevent huge indents.